### PR TITLE
Structure build

### PR DESCRIPTION
**Pull request by Booze33**
- install webpack
- install linters
- Build html structure
- Build basic JavaScript
- Check an fix errors

### CODE REVIEW

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Booze33 ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that `your readme file is not descriptive` please follow this [README template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<body>
+    <section class="box">
+        <header class="top"><h1>Today's To Do </h1></header>
+        <div class="your">
+            <input class="add-input" placeholder="Add to your list..." title="Add list">
+            <button type="submit" class="add">
+                <i class="fa fa-arrow-left" aria-hidden="true"></i>
+            </button>
+        </div>
+        <div class="input"></div>
```
Comment (Tufahel) [inline src/index.html:25]: - [ ] Kindly add the marked items to match the given basic design. You don't need to add any functionality, now.

image

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars */
```
Comment (Tufahel) [inline src/index.js:1]: - [ ] Kindly do not use any `comments` to skip the linter errors, rather fix this in the code. Every linter error has its fix, which may be in the syntax or the format of the code.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Booze33 :wave: 

Well done. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ Gitflow is correct.

## Required Changes ♻️

> Kindly check the inline comments for review.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — COMMENTED** [review]
Hi @Booze33, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you have a live demo section in your README file and to have a professional README file you should remove this section till you have your app deployed so you can just add the live demo section in case you have deployed link for your project is present. .✅❗💯🤩✨✨

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Reem-lab — COMMENTED** [review]
Hi @Booze33, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you have information for a privous project since the title is awesome books and a description for awesome books as well so kindly update all your project information  .✅❗💯🤩✨✨

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @Booze33    , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] kindly  Add a descriptive description in your pull request in future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

**Pull request merged**